app/src/pages/Homepage.tsx

```diff
@@ -5,10 +5,10 @@ import { Alignment, BackgroundView, Box, Direction, IconButton, Image, KibaIcon,
 
 export const HomePage = (): React.ReactElement => {
   const navigator = useNavigator();
-  const [account, setAccount] = React.useState<string>('');
+  const [accountId, setAccountId] = React.useState<string>('');
   const [address, setAddress] = React.useState<string>('');
   const onAccountClicked = () => {
```

Comment:
You don't need callback functions btw, you can use target on the buttons. It's nicer cos people would be able to command+click to open a new tab that way